src/requirements/data/index.ts

```diff
@@ -160,8 +168,13 @@ const json: RequirementsJson = {
       schools: ['AS1', 'AG', 'AS2'],
       requirements: infoRequirements,
     },
+    ISST1: {
+      name: 'Information Science, Systems, and Technology [before Fall 2020]',
+      schools: ['EN'],
+      requirements: oldIsstRequirements,
+    },
     ISST: {
-      name: 'Information Science, Systems, and Technology',
+      name: 'Information Science, Systems, and Technology [Fall 2020 and after]',
```

Comment:
Nit: Could we do `ISST1` for the pre Fall 2020 ISST major and `ISST2` for the Fall 2020 and after ISST major? Just to be consistent with how we're naming `AS1` and `AS2`